### PR TITLE
Update french translation - WebExtensions / Misc

### PR DESCRIPTION
**Pull request by yoriiis**
I made some corrections on these pages:

* `mozilla/add-ons/webextensions/api/runtime/onmessage/index.html`
* `mozilla/add-ons/webextensions/build_a_cross_browser_extension/index.html`
* `mozilla/add-ons/webextensions/your_first_webextension/index.html`

### CODE REVIEW

File: files/fr/mozilla/add-ons/webextensions/build_a_cross_browser_extension/index.html [hunk 1]
```diff
@@ -82,7 +82,7 @@ <h3 id="Gestion_asynchrone_des_événements_API">Gestion asynchrone des événem
 
 <p>Pour plus de détails et d'informations sur l'utilisation du polyfill avec un module bundler, voir le <a href="https://github.com/mozilla/webextension-polyfill/blob/master/README.md">readme du projet sur GitHub.</a></p>
```
Comment (tristantheb) [inline files/fr/mozilla/add-ons/webextensions/build_a_cross_browser_extension/index.html:83]: [Changement] Cet élément est une notice sur la version Anglaise.

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Build_a_cross_browser_extension

File: files/fr/mozilla/add-ons/webextensions/build_a_cross_browser_extension/index.html [hunk 2]
```diff
@@ -82,7 +82,7 @@ <h3 id="Gestion_asynchrone_des_événements_API">Gestion asynchrone des événem
...
 
-<p>Il existe d'autres options de polyfill mais, au moment d'écrire ces lignes, aucune ne fournit la couverture de l'API Polyfill du navigateur WebExtension. Ainsi, lorsque vous n'avez pas choisi Firefox comme premier choix, vos options sont d'accepter les limitations des polyfills alternatifs, de porter sur Firefox et d'ajouter la prise en charge du cross-browser, ou de développer votre propre polyfill.</p>
+<p>Il existe d'autres options de polyfill mais, au moment d'écrire ces lignes, aucune ne fournit la couverture du Polyfill pour l'API WebExtension du navigateur. Ainsi, lorsque vous n'avez pas choisi Firefox comme premier choix, vos options sont d'accepter les limitations des polyfills alternatifs, de porter sur Firefox et d'ajouter la prise en charge du cross-browser, ou de développer votre propre polyfill.</p>
```
Comment (tristantheb) [inline files/fr/mozilla/add-ons/webextensions/build_a_cross_browser_extension/index.html:85]: [ Changement] Cet élément est différent de la version Anglaise
> There are other polyfill options but, at the time of writing, none provide the coverage of the WebExtension browser API Polyfill. So, where you haven’t targeted Firefox as your first choice, your options are to accept the limitations of alternative polyfills, port to Firefox and add cross-browser support, or develop your own polyfill.

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Build_a_cross_browser_extension

**Review by tristantheb — CHANGES_REQUESTED** [review]
Hello :)

There are some elements of the file that do not correspond to the original version (en-US), since you have an open PR request, I invite you to take advantage of it to update your file (styles/texts) thanks to the English version of this one, this last one including design elements and/or texts different from the current and proposed FR version.

Needed for merging:
- Update the current content with the en-US version
- Translate new text/update old text
- Update the HTML content to match the style of the en-US document

Thank you in advance and thank you for the changes provided which are ok for the FR correction 👍🏻

File: files/fr/mozilla/add-ons/webextensions/build_a_cross_browser_extension/index.html (hunk 1 above)
Comment (yoriiis) [reply files/fr/mozilla/add-ons/webextensions/build_a_cross_browser_extension/index.html]: C'est corrigé.

File: files/fr/mozilla/add-ons/webextensions/build_a_cross_browser_extension/index.html (hunk 2 above)
Comment (yoriiis) [reply files/fr/mozilla/add-ons/webextensions/build_a_cross_browser_extension/index.html:95]: Pouvez-vous être plus précis ? Le paragraphe en Français et en Anglais m'ont l'air correct.

**Comment by yoriiis** [comment]
Hi @tristantheb,
I updated the French version from the English one but there are severals texts that are not translated. Maybe it can be done in another PR?

**Review by tristantheb — COMMENTED** [review]
@yoriiis S'il reste des parties en Anglais, quitte à mettre à jour la traduction FR, je pense que tu peux directement finir la traduction dans cette même Requête, finir de traduire un fichier reste dans l'optique de mettre à jour la traduction française ^^

(Pour l'exemple, je suis en train de traduire tout un dossier inexistant du FR, et seulement 3 pages existent, et bien vu que je travail sur un dossier complet la mise à jour de ces pages déjà existante sera dans la même PR, comme ça tout est regroupé en 1 et pas séparé sur plusieurs requêtes)

------
If there are still parts in English, even if you have to update the FR translation, I think you can directly finish the translation in this same Request, finishing the translation of a file remains in the perspective of updating the French translation ^^

(For example, I'm translating a whole file that doesn't exist in the FR, and only 3 pages exist, and since I'm working on a complete file the update of these existing pages will be in the same PR, so everything is grouped in 1 and not separated on several queries)

**Comment by SphinxKnight** [comment]
Je me suis permis de compléter la traduction sur la partie multi-navigateur et de mettre à jour le contenu par rapport aux versions anglaises.
Merci @yoriiis pour cette contribution :)

**Pull request merged**